libs/output-mapping/src/Storage/TableStructureValidator.php

```diff
@@ -14,267 +14,27 @@
 use Keboola\StorageApi\ClientException;
 use Psr\Log\LoggerInterface;
 
-class TableStructureValidator
+class TableStructureValidator extends AbstractTableStructureValidator
 {
-    public function __construct(
-        readonly private bool $hasNewNativeTypeFeature,
-        readonly private LoggerInterface $logger,
-        readonly private Client $client,
-    ) {
-    }
-
-    /**
-     * @param MappingFromConfigurationSchemaColumn[] $schemaColumns
-     */
-    public function validateTable(string $tableId, array $schemaColumns): TableChangesStore
-    {
-        $tableChangesStore = new TableChangesStore();
-
-        if (!$this->hasNewNativeTypeFeature) {
-            return $tableChangesStore;
-        }
-        if (!$schemaColumns) {
-            return $tableChangesStore;
-        }
```

Comment:
tyhle podmínky jsou tady zbytečně, tak jsem je do přepsanýho kódu vůbec nedával

ChangesStore totiž vstupuje jen do StoragePreparer a tam je podmínka stejná, takže tady to řešit nemusíme

https://github.com/keboola/platform-libraries/blob/main/libs/output-mapping/src/Storage/StoragePreparer.php#L40